mbs_results/pivot_imputation_value.py

```diff
@@ -0,0 +1,169 @@
+import pandas as pd
+
+
+def merge_counts(
```

Comment:
I am quite sure counts are included in the post estimate dataframe

Reply:
Thank you - yes we now realise they are. I have removed the merge_counts function and relevant input data.